### PR TITLE
Use newest corelib to fix passkey auth

### PR DESCRIPTION
**Pull request by wojtek2288**
https://github.com/leancodepl/dotnet-kratos-client/pull/4

### CODE REVIEW

File: backend/tests/Examples/ExampleApp.Examples.Tests/Handlers/Booking/AuthorizeWhenOwnsReservationAuthorizerTests.cs [hunk 1]
```diff
@@ -65,7 +65,10 @@ private Task<bool> CheckAsync(CustomerId userId, ReservationId reservationId)
     }
 
     private HttpContext Context(CustomerId userId) =>
-        new DefaultHttpContext { User = new ClaimsPrincipal(new ClaimsIdentity([new("sub", userId.ToString())])) };
+        new DefaultHttpContext
+        {
+            User = new ClaimsPrincipal(new ClaimsIdentity([new Claim("sub", userId.ToString())])),
```
Comment (Saancreed) [inline backend/tests/Examples/ExampleApp.Examples.Tests/Handlers/Booking/AuthorizeWhenOwnsReservationAuthorizerTests.cs:70]: Weird, is there another overload the previous form conflicted with? I don't see any build errors locally.

Comment (wojtek2288) [reply backend/tests/Examples/ExampleApp.Examples.Tests/Handlers/Booking/AuthorizeWhenOwnsReservationAuthorizerTests.cs:70]: I got: `The call is ambiguous between the following methods or properties: 'ClaimsIdentity.ClaimsIdentity(IEnumerable<Claim>?)' and 'ClaimsIdentity.ClaimsIdentity(string?)' [ExampleApp.Examples.Tests]csharp(CS0121)`

Comment (wojtek2288) [reply backend/tests/Examples/ExampleApp.Examples.Tests/Handlers/Booking/AuthorizeWhenOwnsReservationAuthorizerTests.cs]: Huh it must have been some weird local error. I removed this change.

Comment (Saancreed) [reply backend/tests/Examples/ExampleApp.Examples.Tests/Handlers/Booking/AuthorizeWhenOwnsReservationAuthorizerTests.cs]: SDK version mismatch perhaps? :thinking:

**Review by Saancreed — APPROVED** [review]

**Pull request merged**